uc-crux-llvm/src/UCCrux/LLVM/Stats.hs

```diff
@@ -40,7 +40,7 @@ data Stats = Stats
   { missingAnnotation :: !Word,
     symbolicallyFailedAssert :: !Word,
     timeouts :: !Word,
-    truePositiveFreq :: Map TruePositiveTag Word,
+    truePositiveFreq :: Map Text Word,
```

Comment:
Why switch to `Text` as the key type here?

Reply:
Hmm good question... I think it's not necessary. I'll try changing it back!